<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>600</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>BROWSE TRAININGS</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
border: 10px solid #FB98AD;background-color: #0C0C0C;
color: #FB98AD</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="edit_training_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>490</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>EDIT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_training_btn">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>490</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="new_training_btn">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>490</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>NEW</string>
    </property>
   </widget>
   <widget class="QLabel" name="ur_train_lbl">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 20pt &quot;PRIMETIME&quot;;
border: none;</string>
    </property>
    <property name="text">
     <string>YOUR TRAININGS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="main_menu_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>90</y>
      <width>231</width>
      <height>361</height>
     </rect>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
    </property>
   </widget>
   <widget class="QPushButton" name="refresh_btn">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>20</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #1D1D1D;border: 3px solid #FB98AD;
font: 8pt &quot;PRIMETIME&quot;;</string>
    </property>
    <property name="text">
     <string>REFRESH</string>
    </property>
   </widget>
   <widget class="QListWidget" name="training_list_collection">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>291</width>
      <height>361</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
